### PR TITLE
Use semantic-release

### PR DESCRIPTION
**Pull request by dominykas**
I'm deliberately saying that the commits _should_ follow the convention and I'm deliberately not adding a linter for that (just to avoid the burden on any potential contributors).

I'll add a config to determine bump levels based off PR labels, but until then - I'm sure we can manage without enforcing anything, so the goal here is that I can publish this using `npx semantic-release`.

### CODE REVIEW

**Review by rodion-arr — APPROVED** [review]
Ok for me 👍🏻

File: README.md [hunk 1]
```diff
@@ -35,3 +35,8 @@ Example:
   [wiby test](./USAGE.md#wiby-test)    Test your dependents
 
   [wiby result](./USAGE.md#wiby-result) Fetch the results of your tests
+
+## Development
+
+- This repository uses `semantic-release` and commits should follow the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
```
Comment (ljharb) [inline README.md:41]: I’m opposed to two things here: the word “chore” in commits, and automatically publishing. Your OP implies you’re only going to manually publish, so that’s good, but i really don’t like this enforced convention, and i don’t believe semantic-release allows it to be configurable.

Comment (dominykas) [reply README.md:41]: Semantic release does allow it to be configurable, but I don't think it matters, because I don't think it's worth enforcing.

At work we've had massive success with a plugin which determines the bump level off a label on PR, and that has been without any controversy as well (unlike the commit message enforcement, which was a no go from day one) - I just don't have the time to configure that yet.

Comment (dominykas) [reply README.md:41]: I can try to reword this to avoid even the "should"?

Comment (ljharb) [reply README.md:41]: Only the releaser should determine the semver bump - PR reviewers/mergers aren't and shouldn't be the last line of defense. A PR label is more helpful than a commit convention, to be sure.

I like having commit conventions, but I don't think Angular's should be mentioned at all.

Comment (dominykas) [reply README.md:41]: I only mention the Angular convention cause it's the default and with the current volume of contributions not configuring a custom config saves time.

Comment (dominykas) [reply README.md:41]: Can I merge this?

Comment (ljharb) [reply README.md:41]: Is a custom config even possible? If we're not planning on enforcing it, then why mention it?

Comment (dominykas) [reply README.md:41]: Yes, [it is possible](https://github.com/semantic-release/commit-analyzer/#configuration). It doesn't even have to be driven off commit messages (I intend to configure PR labels, which is a bit more versatile).

Mentioning so that the current state is actually documented 🤷‍♂️

Comment (ljharb) [reply README.md:41]: Hmm, seems like we could make a package, and `config` with it relatively easily, and then that advice could go here? (assuming we agree on that package's contents, ofc)

Comment (dominykas) [reply README.md:41]: Yup, that's the plan, just that right now it's a bit lower on my priority list, therefore this.

Comment (ljharb) [reply README.md:41]: I'm not sure I understand the point of using semantic-release if you're not enforcing a commit convention, and if you're manually publishing. Can you help me understand?

Comment (dominykas) [reply README.md:41]: I'm not enforcing it to any drive-by contributors, but I will be following it myself, and I hope others will also follow it, but if they don't - the person making the release can push an empty commit with a message and that's that.

Determining the next bump level is only a part of `semantic-release` - it also creates release notes, posts comments on PRs which are included in the release, etc.

Comment (ljharb) [reply README.md:41]: tbh I'd prefer not mentioning unenforced commit conventions at all.

Comment (dominykas) [reply README.md]: Removed the mention - we can revisit later.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,35 +2,8 @@
...
-* (c) The contribution was provided directly to me by some other
-  person who certified (a), (b) or (c) and I have not modified
-  it.
-
-* (d) I understand and agree that this project and the contribution
-  are public and that a record of the contribution (including all
-  personal information I submit with it, including my sign-off) is
-  maintained indefinitely and may be redistributed consistent with
-  this project or the open source license(s) involved.
+- This repository uses `semantic-release` and commits should follow the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
```
Comment (ljharb) [inline CONTRIBUTING.md:9]: this one too?

Comment (dominykas) [reply CONTRIBUTING.md]: Ooops 🤦  https://github.com/pkgjs/wiby/pull/59/commits/425030c8907c8aca1c3f5236617cf35e914e83e3

**Review by ghinks — APPROVED** [review]

**Comment by dominykas** [comment]
Will merge in a few days - I think the comments aren't blocking?

**Review by ljharb — APPROVED** [review]
Still not a fan of semantic-release and non-humans publishing (nor removing 2FA protections to do it), but all my other concerns have been addressed.
